interested-parties.md

```diff
@@ -51,6 +51,7 @@ Intuit | Alexander Matyushentsev
 Intuit | Mukulika Kapas
 Intuit | Hong Wang
 Intuit | Jesse Suen
+Kubermatic | Waleed Malik
```

Comment:
Thanks @ahmedwaleedmalik — A tip similar to https://github.com/gitops-working-group/gitops-working-group/pull/170#pullrequestreview-780057521
> Hi! Welcome! Could you add your GitHub username next to your name - we had not done this with previous interested parties and then realized we really had no efficient way to be able to reach out to all of you so I want to make sure we start fixing that ;)